src/systems/fem_context.C

```diff
@@ -250,8 +259,16 @@ void FEMContext::some_interior_gradient(unsigned int var, unsigned int qp, Outpu
 
 #ifdef LIBMESH_ENABLE_SECOND_DERIVATIVES
 template<typename OutputType,
+         void (FEMContext::*fe_getter)
+           (unsigned int,
+            FEGenericBase
+              <typename TensorTools::MakeReal
+                <typename TensorTools::DecrementRank
+                  <typename TensorTools::DecrementRank
+                    <OutputType>::type>::type>::type> *&)
```

Comment:
Is it possible to use a `typedef` for the function pointer type?  Seems like you have repeated the same 7 lines of code used from line 843 of the header file...